updates aws_direct_connect role to min viable

privateip opened the pull request:
This commit changes the `aws_direct_connect` role to only implement the
minimium viable resources necessary to enable direct connect.  This
removes the role from creating a vpc and vgw

Commit:
updates aws_direct_connect role to min viable

This commit changes the `aws_direct_connect` role to only implement the
minimium viable resources necessary to enable direct connect.  This
removes the role from creating a vpc and vgw

mtraynham reviewed (approved):
LGTM 👍 

**Edit** Actually, did you also want to update the documentation to include the `vpc` and `vgw` variables?

Commit:
update aws_direct_connect role

This update to the role will now generate all necessary values based on
the name of the fabric to connect to. If the value of fabric is not
provided then `default` is used.
